aws-native-ts-s3-folder/README.md

```diff
@@ -0,0 +1,88 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Host a Static Website on Amazon S3
+
+A static website that uses [S3's website support](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html).
+For a detailed walkthrough of this example, see the tutorial [Static Website on AWS S3](https://www.pulumi.com/docs/tutorials/aws/s3-website/).
+
+Note: Some of the resources are not yet supported by the Native AWS provider, so we are using both the Native
+and Classic provider in this example. The resources will be updated to use native resources as they are
+available in AWS's Cloud Control API.
+
+## Deploying and running the program
+
+Note: some values in this example will be different from run to run.  These values are indicated
+with `***`.
+
+1.  Create a new stack:
+
+    ```bash
+    $ pulumi stack init dev
+    ```
+
+1.  Set the AWS region:
+
+    ```
+    $ pulumi config set aws:region us-west-2
```

Comment:
I prefer setting the `AWS_REGION` environment variable instead since that essentially does it for both providers.

Reply:
Done